ocs_ci/templates/ocs-deployment/provider-mode/ocs_storagecluster_converged.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: ocs-storagecluster
   namespace: openshift-storage
 spec:
-  allowRemoteStorageConsumers: true
```

Comment:
When we merge this to master/stable branch, how it will be with upgrade scenario (where we need to deploy 4.18 from this branch)? (not sure, if it is applicable)